src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -634,102 +628,137 @@ public void testIndicateMissingResponses() {
+        deleteComment("-3-1-0");
+    }
+
+    @Test
+    public void testFeedbackResponseCommentActionsForQuestionsView() {
+
+        ______TS("Question view: Failure case: add an empty feedback response comment using comment modal");
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Open Session");
         resultsPage.displayByQuestion();
-
         resultsPage.loadResultQuestionPanel(1);
-        resultsPage.clickCommentModalButton("-3-1-0");
-        resultsPage.deleteFeedbackResponseCommentInModal("-3-1-0-1");
-        resultsPage.verifyRowMissing("-3-1-0-1");
-        resultsPage.isElementPresent(By.id("showResponseCommentAddForm-3-1-0"));
-        resultsPage.closeCommentModal("-3-1-0");
 
-        resultsPage.clickCommentModalButton("-2-1-0");
-        resultsPage.deleteFeedbackResponseCommentInModal("-2-1-0-1");
-        resultsPage.verifyRowMissing("-2-1-0-1");
-        resultsPage.isElementPresent(By.id("showResponseCommentAddForm-2-1-0"));
-        resultsPage.closeCommentModal("-2-1-0");
+        addCommentToEmptyResponse("-2-1-0");
+
+        ______TS("Question view: Typical case: add new feedback response comments using comment modal");
+
+        addCommentValidToResponse("-2-1-0");
+        addCommentValidToResponse("-3-1-0");
 
-        ______TS("Typical case: add edit and delete successively");
+        ______TS("Question view: Typical case: edit an existing feedback response comment using comment modal");
+
+        editCommentOnResponse("-2-1-0");
+
+        ______TS("Question view: Typical case: edit comment created by different instructors using comment modal");
+
+        resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr2", "Open Session");
+        resultsPage.displayByQuestion();
+        resultsPage.loadResultQuestionPanel(2);
+
+        editCommentGivenByDifferentInstructors("-1-1-0");
+
+        ______TS("Question view: Typical case: delete existing feedback response comments using comment modal");
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Open Session");
         resultsPage.displayByQuestion();
         resultsPage.loadResultQuestionPanel(1);
 
-        resultsPage.clickCommentModalButton("-2-1-0");
-        resultsPage.addFeedbackResponseCommentInCommentModal("showResponseCommentAddForm-2-1-0",
-                "successive action comment");
-        resultsPage.verifyCommentRowContent("-2-1-0-1", "successive action comment", "Teammates Test");
+        deleteComment("-2-1-0");
+        deleteComment("-3-1-0");
+    }
 
-        resultsPage.editFeedbackResponseComment("-2-1-0-1", "edited successive action comment");
-        resultsPage.verifyCommentRowContent("-2-1-0-1", "edited successive action comment",
-                "Teammates Test");
-        resultsPage.clickVisibilityOptionForResponseCommentAndSave("responseCommentRow-2-1-0-1", 1);
+    private void addCommentToEmptyResponse(String commentId) {
```

Comment:
addCommentToEmptyResponse > addCommentToEmptyResponseAndCheckStatusMessage()

Because you are doing two things here.

Same for the following method names.